### DISCUSSION TITLE
How to create a structure?

### DISCUSSION THREAD

**Discussion by ghost** [FabricMC/fabric-api#1376] [Mod Dev Support] [answered]
I have all of my .nbts files. how to create some structures?

**Comment by kanpov** [comment, marked as answer]
https://fabricmc.net/wiki/tutorial:structures

**Comment by kanpov** [comment]
Look at Fabric Wiki before asking questions here

**Reply by ghost** [reply]
I checked, but I couldn't find it - I think it wasn't there in march.
